model/src/main/kotlin/config/PathExclude.kt

```diff
@@ -46,7 +46,9 @@ data class PathExclude(
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val comment: String = ""
 ) {
-    private val glob by lazy { FileSystems.getDefault().getPathMatcher("glob:$pattern") }
+    private val glob by lazy {
+        FileSystems.getDefault().getPathMatcher("glob:${pattern.removePrefix("./")}")
```

Comment:
@fviernau isn't this exactly one of the issues you tried to resolve with the `FileMatcher` class? Would it make sense to also use it here?

Reply:
Hm, I don't recall it 100%. What I do recall was that there were differences between the Ant path matcher the FileMatcher uses now and Java's path matcher.

Regarding this change, I wonder why we should handle './' at all.
Would this mean we also have to handle absolute paths starting with '/'?
What does the '.' of the './' refer to? The current dir?

Reply:
> What does the '.' of the './' refer to? The current dir?

Yes. using `./path` instead of `path` is a common way to explicitly state that the path is relative, not absolute. So IMO we should support that syntax.

Reply:
> Would this mean we also have to handle absolute paths starting with '/'?

Well, as ideally that would come for free and should not require any additional code, I'd say yes. I'd only object if handling of absolute paths would require any special case code.

Reply:
I think absolute path matching is already in because if the regex is `/external/tools/*` the fine `external/tools/x.bin` will be excluded.

Reply:
the problem I see from our discussion is that the regex `./external/tools/*` will match the file `import/conf/external/tools/x.bin`, which is then wrong.

Reply:
> Yes. using `./path` instead of `path` is a common way to explicitly state that the path is relative, not absolute. So IMO we should support that syntax.

My point was that I believe './' makes only sense in context where the current dir is actually defined. Like in a console where you have the current working directory.

Reply:
> I think absolute path matching is already in because if the regex is `/external/tools/*`

Note that we're not dealing with regexes here, but with glob expressions.

Reply:
> My point was that I believe './' makes only sense in context where the current dir is actually defined.

Note that in the JVM context the current dir actually *is* always defined (i.e. it's never undefined), see the `user.dir` [system property](https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html), which means relative paths can always be resolved and also are never undefined.

Reply:
> you tried to resolve with the FileMatcher class? Would it make sense to also use it here?

Yes, for consistency it would be good to align on one matcher class eventually.

> I believe './' makes only sense in context where the current dir is actually defined.

I'd argue in case of a path exclude the current dir is defined by the root directory of the source code the path exclude is applied to.

Reply:
Anyway, we're deviating a bit. What I don't like so much about the propose change is that it hard-codes the Unix syntax for explicit relative paths; we'd need to do the same for Windows paths (`.\path`). To avoid that, I was hoping to be able to use `FileMatcher` already here.

Reply:
> we'd need to do the same for Windows paths (`.\path`). 

I am not sure about `FileMatcher` because this is delegating to `AntPathMatcher`.
Maybe we can just just replace the hardcoded '/' with `File.pathSeparator` ?

Reply:
> Maybe we can just just replace the hardcoded '/' with `File.pathSeparator` ?

That wouldn't work if the path exclude in `.ort.yml` was created by a Windows user using `.\`, but the ORT version reading that file runs under Linux (or vice versa)...